Source/MLXNN/Activations.swift

```diff
@@ -326,8 +326,8 @@ public func hardSwish(_ x: MLXArray) -> MLXArray {
 /// ### See Also
 /// - <doc:activations>
 /// - ``glu(_:axis:)``
-public class GLU: Module, UnaryLayer {
-    let axis: Int
+open class GLU: Module, UnaryLayer {
+    public let axis: Int
```

Comment:
This allows someone to write:

```
class SpecialGLU : GLU {
    func callAsFunction(_ x: MLXArray) -> MLXArray {
        super(x) + 1
    }
}
```

which they could not do before.

For example from the python phi code:

```
class LayerNorm(nn.LayerNorm):
    def __call__(self, x: mx.array) -> mx.array:
        return super().__call__(x.astype(mx.float32)).astype(x.dtype)
```